src/test/java/com/google/api/codegen/testdata/py/py_library.baseline

```diff
@@ -1087,6 +1087,15 @@ class LibraryServiceClient(object):
             return=return_,
         )
 
+    @classmethod
+    def archived_book_path(cls, archive_path, book_id):
+        """Return a fully-qualified archived_book string."""
+        return google.api_core.path_template.expand(
+            'archives/{archive_path}/books/{book_id=**}',
```

Comment:
I am not sure about that `=**` prefix; I have not seen that in format strings before.